doc/example/example_MPIPool.py

```diff
@@ -0,0 +1,71 @@
+"""This file serves as an example how to use MPIPoolEngine
+to optimize across nodes and also as a test for the
+MPIPoolEngine."""
+import pypesto
+import numpy as np
+import scipy as sp
+import pypesto.optimize as optimize
+# you need to manually import the MPIPoolEninge
+from pypesto.engine.mpi_pool import MPIPoolEngine
+# the below is needed for testing purposes.
+from numpy.testing import assert_almost_equal
+
+def setup_rosen_problem(n_starts: int = 10):
+    """sets up the rosenbrock problem and return
```

Comment:
```suggestion
    """Sets up the Rosenbrock problem and returns
```

Reply:
Improving upon that: "Set up and return" --> usually use imperative/infinitive in those one-liner docstrings.

Reply:
Used the infinitive, hope that is gramatically correct.